layouts/_partials/components/analytics/goat-counter.html

```diff
@@ -0,0 +1,17 @@
+{{- with .Site.Params.analytics.goatCounter -}}
+    {{- if .code -}}
+        <script 
+            data-goatcounter="https://{{- .code -}}.goatcounter.com/count"
+            data-goatcounter-settings='
+            {
+                "no_onload": {{- if .noOnload }}true{{- else }}false{{- end -}},
+                "no_events": {{- if .noEvents }}true{{- else }}false{{- end -}},
+                "allow_local": {{- if .allowLocal }}true{{- else }}false{{- end -}},
+                "allow_frame": {{- if .allowFrame }}true{{- else }}false{{- end -}}
+            }
+            '
+            async src="//gc.zgo.at/count.js"></script>
+    {{- else -}}
+        {{- errorf "Missing GoatCounter 'code' configuration. See https://www.goatcounter.com/help/start" -}}
+    {{- end -}}
```

Comment:
The `if-else` logic is not required, you can add a simple `if` at the beginning of the file.

https://github.com/imfing/hextra/blob/cc5884dd2a1fa27fe94cd2a750c07a3020683f4a/layouts/_partials/components/analytics/umami.html#L8-L10

You can also use a link to https://imfing.github.io/hextra/versions/latest/docs/guide/configuration/#goatcounter-analytics instead of the GoatCounter doc.